__tests__/app.test.js

```diff
@@ -12,22 +13,61 @@ beforeEach(() => {
   return seed(testData);
 });
 
-describe("far-han-news tests", ()=> {
-    describe("/api/topics", () => {
-        test("GET 200: should return with an array of topics", () => {
-            return request(app).get("/api/topics").expect(200).then(({body})=> {
-                const {topics} = body
-                expect(topics).toEqual(expect.any(Array))
-                expect(topics.length).toBe(3)
-                topics.forEach((topic)=>{
-                    expect(topic).toHaveProperty("slug", expect.any(String))
-                    expect(topic).toHaveProperty("description", expect.any(String))
-                })
-            })
-        })
-        test("GET 404: should return an error for a path that does not exist", () => {
-            return request(app).get("/api/teepics").expect(404)
-
-        })
-    })
-})
\ No newline at end of file
+describe("far-han-news tests", () => {
+  describe("/api/topics", () => {
+    test("GET 200: should return with an array of topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          expect(topics).toEqual(expect.any(Array));
+          expect(topics.length).toBe(3);
+          topics.forEach((topic) => {
+            expect(topic).toHaveProperty("slug", expect.any(String));
+            expect(topic).toHaveProperty("description", expect.any(String));
+          });
+        });
+    });
+    test("GET 404: should return an error for a path that does not exist", () => {
+      return request(app).get("/api/teepics").expect(404);
+    });
+  });
+  describe("/api/artiles/:article_id", () => {
```

Comment:
Fix the typo! I know it's picky and it isn't having an effect on functionality, but it is your portfolio piece! 😉